src/wazuh_db/wdb.h

```diff
@@ -833,6 +833,31 @@ int wdb_parse_ciscat(wdb_t * wdb, char * input, char * output);
 int wdb_parse_sca(wdb_t * wdb, char * input, char * output);
 
 
+/**
+ * @brief Function to parse get operation over the sys_osinfo database table.
+ *
+ * @param wdb The Global struct database.
+ * @param output buffer output, on success responses are:
```

Comment:
Doxygen style: Use a capital letter to start the param description: "_Buffer output,_"

Reply:
Fixed